### PR TITLE
Fix/try redict

### PR DESCRIPTION
**Pull request by mfocko**

### CODE REVIEW

File: docs/deployment/specifics/redict.md
Comment (lbarcziova) [inline docs/deployment/specifics/redict.md:1]: thanks for writing this down!

**Review by nforro — APPROVED** [review]

File: docs/deployment/specifics/redict.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Redict
+---
+
+## Seamless migration between providers
+
+We have tested a seamless migration from Redis to Redict on our production
+deployment. To reproduce:
+
+1. We have deployed Redict to our production cluster.
```
Comment (lbarcziova) [inline docs/deployment/specifics/redict.md:10]: maybe worth adding a command how was this done (`make deploy redict`?)

**Review by lbarcziova — APPROVED** [review]
nice!

**Review by majamassarini — APPROVED** [review]
Thanks!

File: docs/deployment/specifics/redict.md (hunk 1 above)
Comment (mfocko) [reply docs/deployment/specifics/redict.md:10]: Now that you mention it, I forgot the check for deploying both at the same time :D

**Pull request merged**